package.json

```diff
@@ -72,6 +72,7 @@
     "react-redux": "^7.2.6",
     "react-select": "^4.3.1",
     "react-select-fast-filter-options": "^0.2.3",
+    "react-simple-star-rating": "^4.0.5",
```

Comment:
You should also commit the change in `yarn.lock`  file.
If you did not install with yarn (`yarn add react-simple-star-rating`)  you might have to run the command again to generate the changes to the lockfile.